### PR TITLE
Env var case

### PR DESCRIPTION
**Pull request by abadger**
##### SUMMARY
Changelog for https://github.com/ansible-collections/community.general/pull/377

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/fix-oc-conn-plugin-envvar.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+bugfixes:
```
Comment (felixfontein) [inline changelogs/fragments/fix-oc-conn-plugin-envvar.yml:1]:
```suggestion
breaking_changes:
```

File: plugins/connection/oc.py [hunk 2]
```diff
@@ -85,7 +85,7 @@
         vars:
           - name: ansible_oc_context
         env:
-          - name: k8S_AUTH_CONTEXT
+          - name: K8S_AUTH_CONTEXT
```
Comment (felixfontein) [inline plugins/connection/oc.py:88]: I'm wondering why this change is shown again.

Comment (abadger) [reply plugins/connection/oc.py:88]: I based it off the old branch without pulling the latest upstream master... I thought git/github would be smart enough to resolve that (it would on merge, I think).  Now I'm paranoid, though... /me updates master and rebases.

File: changelogs/fragments/fix-oc-conn-plugin-envvar.yml (hunk 1 above)
Comment (abadger) [reply changelogs/fragments/fix-oc-conn-plugin-envvar.yml]: Force pushed and lost this, But I caught it... This change should be restored.

File: changelogs/fragments/fix-oc-conn-plugin-envvar.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+breaking_changes:
+- The environment variable for the auth context for the oc.py connection plugin
+  has been corrected (K8S_CONTEXT).  It was using an initial lowercase k by mistake.
+  https://github.com/ansible-collections/community.general/pull/377
```
Comment (felixfontein) [inline changelogs/fragments/fix-oc-conn-plugin-envvar.yml:4]:
```suggestion
  (https://github.com/ansible-collections/community.general/pull/377).
```

File: plugins/connection/oc.py (hunk 2 above)
Comment (felixfontein) [reply plugins/connection/oc.py]: I was thinking that github would be smart enough too... That's why I was so confused when I saw the change again. But apparently it is not.
